src/app/assets/scss/views/river-detail/_river-creator.scss

```diff
@@ -0,0 +1,32 @@
+.river-creator {
+  .form-fields-column,
+  .map-column {
+    padding-top: 1rem;
+    order: 2;
+    @include carbon--breakpoint("sm") {
+      min-height: calc(80vh - 50px);
```

Comment:
why the min/max double set as opposed to just setting the height? probably some trick I'm just not seeing...